Changed controller function for POST request

hQian2718 opened the pull request:
the controller function createResponse now correctly destructures the incoming survey JSON.

MrPentagon404 committed:
Changed controller function for POST request

the controller function createResponse now correctly destructures the incoming
survey JSON.

MrPentagon404 committed:
Added email and name to document

Updated Schema again to specify everything is string. Added fields for emails
and full name.

yutianrun7 reviewed (approved):
polish backend question storage and fit that to the received frontend data structure

MrPentagon404 committed:
GET requests working

Created functions that search through the database. Can return all survey
responses, sorted by user email addresses, or search for one response.
If searches fail return 404, if other errors encountered return 500.
